File: python/armstrong-numbers/armstrong_numbers.py
def is_armstrong_number(number):
    try: 
        number_of_digits=len(str(number))
        digits = str(number)
        return sum(int(i) ** number_of_digits for i in digits) == number
    except Exception as e:
        raise Exception("Something bad is happening!!")
    pass

File: javascript/armstrong-numbers/armstrong-numbers.js

export const isArmstrongNumber = (number) => {
	number_length = number.toString().length
	if ( number_length == 1 )
		return true
	if ( number_length > 2 ){
		for (i = 0; i < 5; i++) {
			text += "The number is " + i + "<br>";
		}	
	}
	return false
};
